proto/brpb.proto

```diff
@@ -184,6 +184,36 @@ message BackupRequest {
     CipherInfo cipher_info = 14;
 }
 
+message StreamBackupTaskInfo {
+    // The storage for backup, parsed by BR.
+    StorageBackend storage = 1;
+    // The time range for backing up.
+    uint64 start_ts = 2;
+    uint64 end_ts = 3;
```

Comment:
will tikv remove this task if `resolved_ts` reached `end_ts`?

Reply:
I guess we can leave it a "done" status and let the user to delete?

Reply:
The Task would be exist, but the observing of stream-log has been stoped. And user can status/stop the task in this situation ?